project/src/main/webapp/index.html

```diff
@@ -1,18 +1,50 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8" />
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
+
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="restuarant-prefs">
+      <label for="cuisine">What cuisine?</label>
+      <select name="cuisine" id="cuisine">
+        <option value="Italian">Italian</option>
+        <option value="Mexican">Mexican</option>
+        <option value="India">Indian</option>
+      </select>
+      <br />
```

Comment:
nit: don't close void elements, `<br>` instead of `<br />` 
https://google.github.io/styleguide/htmlcssguide.html#Document_Type

Reply:
For this and the ones below.